### PR TITLE
Adding Quarkus 'getting-started' devfile

### PR DESCRIPTION
**Pull request by ibuziuk**
Related issue - https://github.com/eclipse/che/issues/17961
Try it on Hosted Che:

[Contribute](https://che.openshift.io/f?url=https://raw.githubusercontent.com/ibuziuk/devfile/che-17961/getting-started/quarkus/devfile.yaml)

image

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: getting-started/quarkus/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        port: 8080
+        attributes:
+          path: /hello/greeting/che-user
+
+  - type: dockerimage
+    alias: ubi-minimal
+    image: 'registry.access.redhat.com/ubi8/ubi-minimal'
+    memoryLimit: 32M
+    mountSources: true
+    endpoints:
```
Comment (benoitf) [inline getting-started/quarkus/devfile.yaml:46]: it should contain the debug port 5005 with public: false

File: getting-started/quarkus/devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+          -Duser.home=/home/user"
+      - name: MAVEN_OPTS
+        value: $(JAVA_OPTS)  
+    memoryLimit: 4927M
+    mountSources: true
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    endpoints:
```
Comment (benoitf) [inline getting-started/quarkus/devfile.yaml:35]: it should contain the debug port 5005 with public: false

Comment (benoitf) [reply getting-started/quarkus/devfile.yaml:35]: https://github.com/eclipse/che/issues/18138

Comment (ibuziuk) [reply getting-started/quarkus/devfile.yaml:35]: @benoitf probably should be first fixed in the devfile registry first - https://github.com/eclipse/che-devfile-registry/tree/master/devfiles/quarkus 
Will follow up on the eclipse/che#18138

Comment (benoitf) [reply getting-started/quarkus/devfile.yaml:35]: sure, just wanted to raise awareness

**Pull request merged**